src/Dialogs/MapItemListDialog.cpp

```diff
@@ -251,6 +252,7 @@ MapItemListWidget::OnAction(int id)
   switch (id) {
   case SETTINGS:
     ShowMapItemListSettingsDialog();
+    Profile::Save();
```

Comment:
Why?

Reply:
the settings are elements of the profile.